### PR TITLE
fix(passport validations):

### PR DESCRIPTION
**Pull request by daniel7764**
User can now type in passport by this condition: An alphanumeric string with 10 characters or A numeric string and a possible "/".

### CODE REVIEW

File: client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts [hunk 1]
```diff
@@ -37,15 +37,28 @@ const isIdValid = (id: string | null | undefined) => {
...
      *  note : this is a funcion and not a constant because the logic
      *        will most likely be changed to something more complicated (like id)
      */
-    if (Boolean(id)) {
-        return passportValidation.test(String(id));
+    if(id) {
+        if (id.length === passportLength) {
+            return passportValidation.test(String(id));
+        } else if (id.length === visaLength) {
+            if(id.split('/').length < 3) {
```
Comment (y-golde) [inline client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts:54]: maybe export to const/func named `hasMoreThanOneSlash`

File: client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts [hunk 2]
```diff
@@ -37,15 +37,28 @@ const isIdValid = (id: string | null | undefined) => {
...
+                return visaValidation.test(String(id));
+            } else {
+                return false;
+            }
```
Comment (y-golde) [inline client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts:58]: this else is redundant

File: client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts (hunk 1 above)
Comment (ItayBenMoshe) [inline client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts:54]: Maybe export 3 to const

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts (hunk 1 above)
Comment (daniel7764) [reply client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts]: fixed

Comment (daniel7764) [reply client/src/Utils/auxiliaryFunctions/auxiliaryFunctions.ts]: fixed

**Review by ItayBenMoshe — APPROVED** [review]

**Review by y-golde — APPROVED** [review]

**Pull request merged**